docs/release-notes/8.3.asciidoc

```diff
@@ -14,6 +87,78 @@
 == 8.3.0
 
+[discrete]
+[[known-issue-8.3.0]]
+==== Known issue
+
+8.3.0 has a bug where detection rules that were created or edited in 8.2 will stop running after you upgrade. Because of this, we advise against upgrading from 8.2 to 8.3.0. If you want to upgrade, _we highly recommend that you track your active rules beforehand so you can easily find and re-enable them after you upgrade_. Use the <<rules-api-find, Find rules API>> to capture a list of enabled rules before you upgrade.
+
+NOTE: Use curl or another HTTP tool to run {elastic-sec} API commands.
+
+Here is a curl command that you can run to find enabled rules:
+
+[source,console]
+--------------------------------------------------
+GET /api/detection_engine/rules/_find?per_page=10000&filter=alert.attributes.enabled:true
+--------------------------------------------------
+
+If you already upgraded from 8.2 to 8.3.0, detection rules affected by the bug will have stopped running with an error that is similar to the following example:
+
+[source,text]
+----
+<rule-type>:<UUID>: execution failed - security_exception: [security_exception] Reason: missing authentication credentials for REST request [/_security/user/_has_privileges], caused by: ""
+----
+
+You will need to disable, then re-enable affected rules to restore them to a functional state. These actions generate a new API key using the credentials of the user performing these actions. The rules' states are also reset.
+
+NOTE: To restore failed custom and prebuilt rules, you need privileges to <<enable-detections-ui, manage rules>>.
+
+*Restore custom rules*
+
+Follow these steps:
+
+. Go to the *Rules* page (*Detect -> Rules*) and click *Custom rules*.
+. Sort the rules table by the *Enabled* column to show enabled rules.
+. Select the affected rules, then click *Bulk actions -> Disable*.
++
+NOTE: Affected rules will have a `Failed` execution status and the aforementioned error message in the rule's details.
+
+. Select the affected rules, then click *Bulk actions -> Tags -> Add Tags*. Add a descriptive tag, for example `rules_to_fix`.
+. Re-select the same rules, then click *Bulk actions -> Enable*.
+. Add a descriptive tag, for example `rules_to_fix`. Adding a tag will generate new API keys and solve the issue.
+. Follow these steps if you don't want to wait until the next scheduled rule execution (optional):
+.. From the rules table, filter for the tag you just applied to your custom rules.
+.. Click *Select all x rules* above the rules table.
+.. Click *Bulk actions -> Disable*.
+.. Filter the rules table for the same tag, re-select the same rules, and then click *Bulk actions -> Enable*.
+
+Once the rule runs, the *Last Response* value for the rule will change to `Pending`, and then to `Active` or `OK`.
+
+*Restore prebuilt rules*
+
+Follow these steps:
+
+. Use the <<rules-api-find, Find rules API>> to capture a list of enabled rules before you upgrade. Here is a curl command that you can run to find enabled rules:
```

Comment:
Adding this word to make identical the same passage in the 8.3.1 section -- ignore if they don't need to match exactly.
```suggestion
. Use the <<rules-api-find, Find rules API>> to capture a list of enabled rules before you upgrade. Here is a curl command that you can run to find enabled prebuilt rules:
```